api/database/DBConfig.js

```diff
@@ -0,0 +1,29 @@
+const mysql = require("mysql2");
+
+const config = {
+  db: {
+    /* do not put password or any sensitive info here, done only for demo */
+    host: process.env.DB_HOST || "34.85.164.225",
```

Comment:
What is this hardcoded ip address pointed to? Is this the same for all versions or just specific to you?

Reply:
This is the host of our DB

Reply:
ah I see, thanks for clarifying!